<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::report::ReportProgressDialog</class>
 <widget class="QDialog" name="tt3::report::ReportProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generating report</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-report.qrc">
    <normaloff>:/tt3-report/Resources/Images/Reports/ReportLarge.png</normaloff>:/tt3-report/Resources/Images/Reports/ReportLarge.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="generatingLabel">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="text">
      <string>Generating:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="generatingProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="writingToLabel">
     <property name="text">
      <string>Writing to:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QProgressBar" name="writingToProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-report.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::report::ReportProgressDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::report::ReportProgressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
